stories/accordion/subcomponents/index.stories.tsx

```diff
@@ -134,44 +134,75 @@ export const Truncate = () => {
 
 Truncate.id = `${ACCORDION_SUBCOMPONENTS_STORY_ID_PREFIX}truncate`;
 
-export const Container = () => (
-  <Accordion.Container>
-    <Accordion
-      onClick={() => undefined}
-      title={
+export const Container = () => {
+  const [isOpen, setIsOpen] = useState(true);
+  const onClick = () => setIsOpen(!isOpen);
+
+  return (
+    <Accordion.Container>
+      <Accordion
+        onClick={onClick}
+        title={
+          <Accordion.Content>
+            This Accordion styled with an Accordion.Container parent component
+          </Accordion.Content>
+        }
+        isOpen={isOpen}
+      >
         <Accordion.Content>
           This Accordion styled with an Accordion.Container parent component
         </Accordion.Content>
-      }
-      isOpen
-    >
-      <Accordion.Content>
-        This Accordion styled with an Accordion.Container parent component
-      </Accordion.Content>
-    </Accordion>
-  </Accordion.Container>
-);
+      </Accordion>
+    </Accordion.Container>
+  );
+};
 
 Container.id = `${ACCORDION_SUBCOMPONENTS_STORY_ID_PREFIX}container`;
 
-export const Content = () => (
-  <Accordion.Container>
-    <Accordion
-      onClick={() => undefined}
-      title={<Accordion.Content>Title</Accordion.Content>}
-      isOpen={false}
-    >
-      <Accordion.Content>Expansion with content padding</Accordion.Content>
-    </Accordion>
-    <Accordion
-      onClick={() => undefined}
-      title={<Accordion.Content>Title (open)</Accordion.Content>}
-      isOpen
-    >
-      <Accordion.Content>Expansion with content padding</Accordion.Content>
-    </Accordion>
-  </Accordion.Container>
-);
+export const Content = () => {
```

Comment:
The diff in this file is just making these two stories interactive.

Reply:
useReducer 🙌

Reply:
Yeah I originally set it up when it was 3 separate accordions but kept it after reducing it to 2 because I'd like for our code to use more examples--we use a fair amount of coupled `useState` functions and it might be nice to get more comfortable with other hooks API.